plugins/puma/puma_

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
The only use of a bashism seems to be the array, which could be even integrated into the `awk` that is already spawned (as you already did in the end of the plugin).

Then you could even use a direct `/bin/sh` which is usually lighter (until `sh` is symlinked to `bash`)